### ISSUE TITLE
connect two vehicles via (small) rope or chain.

### ISSUE THREAD

**Issue by Lamandus** [CleverRaven/Cataclysm-DDA#25678]
I am not sure if someone already had the idea, if so, I am sorry to waste your time. This idea came to me while dragging one shopping cart and having it full to the brim. I came around a store with another cart which would be useful. In real life (where the apocalypse is your commune) I would use a chain or rope to connect those still separate vehicles and drag them around while the 2nd one follows suit one tile next. I would ingame use the "jumping cable"system which seems to have the connection basics in.
Bonus: you can use the system to drag your shopping card behind your car. Or even build trailers which can be connected (though the phisics could be wonky)

**Comment by mlangsdorf** [comment]
It's been suggested before. We'd love to implement it.

Known issues:
1. The grab code is finicky and isn't a good base for doing much else.
2. Jumper cables already don't work well with moving vehicles.
3. The collision code is a mess that needs to be cleaned up before we can implement trailers so that towed vehicles don't collide with towing vehicles.

In the short-ish term, bike racks (#25191) will let you connect a single tile wide vehicle to another vehicle, so you'd be able to add the second grocery cart to your cart or truck.

**Comment by Lamandus** [comment]
Then sorry for wasting your time. And thank you for the info

**Comment by mlangsdorf** [comment]
You're not really wasting our time. Thanks for submitting the issue. Closing now as a duplicate of long term plans in the extended vehicle tune up project.
